### ISSUE TITLE
Avatar a11y: Active indicator is not visible in high contrast

### ISSUE THREAD

**Issue by behowell** [microsoft/fluentui#21352]
Active indicator in high contrast should have a visible ring around it:
![image](https://user-images.githubusercontent.com/48106640/150334686-be1e6147-0f18-4ad0-984d-371726d727f9.png)

For reference, here is what it looks like in light theme:
![image](https://user-images.githubusercontent.com/48106640/150334583-70102c86-0b28-4d52-a85f-272a03bf47ec.png)

**Comment by behowell** [comment]
It turns out this is only an issue with the fake high contrast theme used in the screener tests, and not an issue with the actual system high contrast mode.
